docs/content/flink/procedures.md

```diff
@@ -757,13 +757,15 @@ All available procedures are listed below.
    <tr>
       <td>rescale</td>
       <td>
-         CALL [catalog.]sys.rescale(`table` => 'identifier', `bucket_num` => bucket_num, `partition` => 'partition')
+         CALL [catalog.]sys.rescale(`table` => 'identifier', `bucket_num` => bucket_num, `partition` => 'partition', `source_parallelism` => 'source_parallelism', `sink_parallelism` => 'sink_parallelism')
```

Comment:
scan.parallelism and sink.parallelism.

Just be consistent with Flink Connector options.

Reply:
No need. This is a procedure argument, not an option.

Reply:
Can we maintain consistency, why not?